UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneExporter/ExporterTextures.cs

```diff
@@ -86,6 +90,18 @@ private void ExportMetallicGlossTexture(Texture2D texture, string outputPath, bo
 			WriteRenderTextureToDiskAndRelease(destRenderTexture, outputPath, true);
 		}
 
+		/// <summary>
+		/// This exports a Unity cubemap texture to an equirectangular map
+		/// </summary>
+		/// <param name="texture">Unity's cubemap texture to be exported</param>
+		/// <param name="outputPath">The location to export the texture</param>
+		private void ExportCubeMapTexture(Cubemap texture, string outputPath)
+		{
+			var destRenderTexture = RenderTexture.GetTemporary(texture.width * 2, texture.height, 24);
+			Graphics.Blit(texture, destRenderTexture, new Material(Shader.Find("Hidden/CubemapToEquirectangular")));
+			WriteRenderTextureToDiskAndRelease(destRenderTexture, outputPath, false);
```

Comment:
Is there a reason for not using `RenderTexture.ConvertToEquirect`?

Reply:
Also, this will need to most likely handle LDR and HDR differently.

Reply:
> Is there a reason for not using `RenderTexture.ConvertToEquirect`?

It's been a second since I worked on this, but I think it came down to the texture encoding? I do remember trying it and the output result was off from the RGBM encoded format I was expecting. I can try again and see what the issue is.

> Also, this will need to most likely handle LDR and HDR differently.

Yeah, currently I'm only supporting RGBM, this function just takes the input cubemap texture, transforms it to equirect, blits it, and writes it directly to a file. So what goes in comes out in the same way. I see you have HDR texture export now, but at the time I was thinking we should stick with the existing formats meaning RGBM encoded into a PNG. Also it's a bit cheaper to do so.

Ideally the final extension (an EXT or KHR one instead of our MX vendor extension) would use a compressed HDR texture, but I'm still not aware of a way to do that with Basis.

Reply:
Understood. I'm not sure the enum should be called "Cubemap" then. 
Sounds like there's actually enum flags at this point:
- Cubemap vs. 2D
- Linear vs. sRGB
- HDR vs. LDR vs. RGBM